index.js

```diff
@@ -0,0 +1 @@
+module.exports = require('./.tsconfig.json');
```

Comment:
[suggestion] I might recommend relying on the `main` and `files` properties in package.json ([example](https://github.com/sindresorhus/tsconfig/blob/7b91ff96081e52a8b97c75c30029807db061b670/package.json#L16-L18)) to define which files get published to NPM and to treat `tsconfig.json` as the entry point for the module instead of "index.js" (AFAIK, TypeScript configs may only be defined as JSON files so I'm a bit concerned that treating a JS file as the entry point for `@edx/typescript-config` may not work with TypeScript).